README.md

```diff
@@ -87,7 +87,60 @@ Unlike many authorization systems where a service controls access to resources i
+│   ┌──────────────────────────────────────────────────────────┐  ┌──────────────────────┐   │
+│   │                                                          │  │                      │   │
+│   │  Principal                                               │  │  Validator           │   │
+│   │                                                          │  │                      │   │
+│   │  ┌──────────────────────────┐                            │  │                      │   │
+│   │  │                          │                            │  │                      │   │
+│   │  │  Signer                  │                            │  │                      │   │
+│   │  │                          │                            │  │                      │   │
+│   │  │  ┌────────────────────┐  │  ┌──────────────────────┐  │  │                      │   │
+│   │  │  │                    │  │  │                      │  │  │                      │   │
+│   │  │  │  Issuer            │  │  │  Audience            │  │  │                      │   │
+│   │  │  │                    │  │  │                      │  │  │                      │   │
+│   │  │  │  ┌──────────────┐  │  │  │                      │  │  │                      │   │
+│   │  │  │  │              │  │  │  │                      │  │  │                      │   │
+│   │  │  │  │  Owner       │  │  │  │                      │  │  │                      │   │
+│   │  │  │  │              │  │  │  │                      │  │  │                      │   │
+│   │  │  │  └──────────────┘  │  │  │                      │  │  │                      │   │
+│   │  │  │                    │  │  │                      │  │  │                      │   │
+│   │  │  │  ┌──────────────┐  │  │  │                      │  │  │                      │   │
+│   │  │  │  │              │  │  │  │                      │  │  │                      │   │
+│   │  │  │  │  Revoker     │  │  │  │                      │  │  │                      │   │
+│   │  │  │  │              │  │  │  │                      │  │  │                      │   │
+│   │  │  │  └──────────────┘  │  │  │                      │  │  │                      │   │
+│   │  │  │                    │  │  │                      │  │  │                      │   │
+│   │  │  └────────────────────┘  │  │                      │  │  │                      │   │
+│   │  │                          │  │                      │  │  │                      │   │
+│   │  └──────────────────────────┘  └──────────────────────┘  │  │                      │   │
+│   │                                                          │  │                      │   │
+│   └──────────────────────────────────────────────────────────┘  └──────────────────────┘   │
+│                                                                                            │
+└────────────────────────────────────────────────────────────────────────────────────────────┘
+```
```

Comment:
@hugomrdias & @Gozala I based this on a combination of what's in the Discord and comments on this PR. Does this more closely match your picture?

Reply:
Preview: https://github.com/ucan-wg/spec/blob/roles/README.md#21-roles

Reply:
This looks good to me! Owner covers the scenario I described in my comment about. 👌

Reply:
@expede this looks awesome really helpful